src/Samples/Samples.sln

```diff
@@ -17,8 +17,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ImplementNotifyPropertyChan
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MakeConstCS", "CSharp\MakeConst\Impl\MakeConstCS.csproj", "{DE9D901F-43D6-4559-A068-B6E28E04F12E}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MakeConstCS.UnitTests", "CSharp\MakeConst\Test\MakeConstCS.UnitTests.csproj", "{BFD0109A-767E-41F6-8EF3-1A75FB4F5E8D}"
```

Comment:
MakeConstCS tests have not been run in... *One Thousand Years*.  If they provide any benefit, we've somehow learned to live without it